integration/resources/compose/consul_catalog.yml

```diff
@@ -2,7 +2,9 @@ consul:
   image: consul:1.6.2
   ports:
     - 8500:8500
-  command: "agent -server -bootstrap -ui -client 0.0.0.0"
+  command: "agent -config-file=/server.hcl"
+  volumes:
+    - ../consul/server.hcl:/server.hcl
```

Comment:
Why did you need to move to a config file? Not possible to enable connect on the command line?

Reply:
I'll double check. One cannot do that specifically for connect but consul has:
```
  -hcl=<value>
     hcl config fragment. Can be specified multiple times.
```
Whether that will be more readable is questionable though. And yes the move to the host for those tests is a result of bind mounting the config file.

Reply:
Pushed a new commit that puts it back inline; if we need to adjust more options though that might become unreadable :D